src/playwright-runner.js

```diff
@@ -286,7 +293,20 @@ async function run (nodeBin, runCfgPath, suiteName) {
 
   const excludeParams = ['screenshot-on-failure', 'video', 'slow-mo'];
 
+  let usePlaywrightCfg = false;
+  const defaultCfgFiles = ['./playwright.config.ts', './playwright.config.js'];
+  for (const file of defaultCfgFiles) {
+    if (fs.existsSync(path.join(projectPath, file)) === true) {
+      usePlaywrightCfg = true;
+    }
+  }
   for (let [key, value] of Object.entries(args)) {
+    // There is a conflict if the playwright project has a `browser` defined,
+    // since the job is launched with the browser set by saucectl, which is now set as the job's metadata.
+    if (usePlaywrightCfg && key === 'browser') {
```

Comment:
There's already some logic around skipping certain keys, see `excludeParams` above: https://github.com/saucelabs/sauce-playwright-runner/pull/131/files#diff-666450f3c804e34e85768192da8dae7fd947b4b14db90b48f12cea82711fcf22R294

We can keep the exclusion logic together by adding `browser` to the `excludeParms` list if `usePlaywrightCfg === true`